Tasks/WindowsMachineFileCopy/task.json

```diff
@@ -63,22 +63,30 @@
             "label": "Clean Target",
             "defaultValue": "false",
             "required": false,
-            "helpMarkDown": "Setting it to true will clean up the destination folder before copying the files."
+            "helpMarkDown": "Setting it to true will clean the destination folder before copying the files."
         },
         {
-            "name": "CopyFilesInParallel",
+            "name": "DeployFilesInParallel",
             "type": "boolean",
-            "label": "Copy Files In Parallel",
+            "label": "Copy Files in Parallel",
             "defaultValue": "true",
             "required": false,
             "groupName":"advanced",
-            "helpMarkDown": "Setting it to true will copy files in parallel on the target machines."
+            "helpMarkDown": "Setting it to true will copy files in parallel to the machines."
         }
     ],
-    "instanceNameFormat": "Copy files on machine group $(EnvironmentName)",
+    "sourceDefinitions": [ 
+     { 
+         "target": "EnvironmentName", 
+         "endpoint": "/$(system.teamProject)/_apis/vslabs/environments", 
+         "selector": "jsonpath:$.value[*].name", 
+         "authKey": "tfs:DevTestLabs" 
+     } 
+    ], 
+    "instanceNameFormat": "Machine Group: $(EnvironmentName)",
 	 "execution": {
         "PowerShell": {
-            "target": "$(currentDirectory)\\CopyFilesToMachines.ps1",
+            "target": "$(currentDirectory)\\WindowsMachineFileCopy.ps1",
             "argumentFormat": "",
```

Comment:
Changes look fine. Make sure to check-in pester test changes, E2E test changes in a single PR at the same time when you commit this change into master.